src/searchers_compare.py

```diff
@@ -0,0 +1,212 @@
+import pytest
+import os
+import json
+from typing import Union
+
+import numpy as np
+
+from jina import __version__
+from jina.types.arrays.memmap import DocumentArrayMemmap
+from jina import Document, Flow, Executor, requests, DocumentArray
+
+from statistics import mean, stdev
+from pympler import asizeof, tracker
+from .utils.timecontext import TimeContext
+
+NUM_REPETITIONS = 5
+NUM_QUERY_DOCS_PER_REQUEST = 64
```

Comment:
Would it make sense to test it with smaller Resquest size for simulating 'real time use'?
Maybe add this as a fixture rather than a global variable and benchmark with  `[1,5,64]`.
To me 64 seems still a rather large value.

Reply:
Can u review again?

Reply:
I've seen the update without global variable and  request in size [1, 32, 64].
Looks good.